porting/debug-build/wifi.rst

```diff
@@ -11,18 +11,24 @@ Testing wifi is done using NetworkManager
 
    nmcli d
 
-Qcom devices
-------------
+Qcom devices and CAF devices
+------------------------------------
```

Comment:
the line length should be equal to the text length

Reply:
it's related to almost all of your PRs. it doesn't break anything, but it looks horrible and makes me suffer